robottelo/ui/hosts.py

```diff
@@ -97,7 +99,8 @@ def create(self, arch=None, cpus='1', cv=None, custom_ptable=None,
                lifecycle_env=None, loc=None, mac=None, media=None,
                memory='768 MB', name=None, org=None, os=None, ptable=None,
                puppet_ca=None, puppet_master=None, reset_puppetenv=True,
-               resource=None, root_pwd=None, subnet=None):
+               resource=None, root_pwd=None, subnet=None, network_type=None,
+               network=None):
```

Comment:
we may need to consider sending a list here for future.  Not a blocker for this pr.